src/css/components/Navigation.css

```diff
@@ -14,6 +14,12 @@
   border: 1px solid rgb(143, 142, 142);
   flex-grow: 1;
 }
+
+.not-selected:hover {
```

Comment:
Remember that there is no `:hover` on mobile devices, so make sure you’re not relying on that behavior for anything essential.